compiler/src/typed/oprint.re

```diff
@@ -371,21 +371,13 @@ and print_out_type_1 = ppf =>
   | ty => print_out_type_2(ppf, ty)
 and print_out_type_2 = ppf =>
   fun
-  | Otyp_tuple(tyl) => {
-      pp_open_box(ppf, 1);
-      pp_print_char(ppf, '(');
-      fprintf(
-        ppf,
-        "@[<0>%a@]",
-        print_typlist(print_simple_out_type, ","),
-        tyl,
-      );
-      if (List.length(tyl) == 1) {
-        pp_print_char(ppf, ',');
-      };
-      pp_print_char(ppf, ')');
-      pp_close_box(ppf, ());
-    }
+  | Otyp_tuple(tyl) =>
+    fprintf(
+      ppf,
+      "@[<0>(%a)@]",
+      print_typlist(print_simple_out_type, ","),
+      tyl,
+    )
```

Comment:
This is fine but we should definitely land the removal of single argument tuples before we merge this.

Reply:
Yep! Since enum constructors are now record or tuple, we need to remove those before this works.